HOST lewis.lbl.gov
only_i 82
only_code None
force_amber_prep True
dry_run False
1 1z3z,A,2.900,1zod,A,1.800
2 1psi,A,2.920,3ndd,A,1.500
3 3bpo,A,3.000,3l5w,I,2.000
4 1e6j,P,3.000,3mge,A,1.900
5 1r46,A,3.250,3hg3,A,1.900
6 2j0g,A,2.850,2j1g,A,1.950
7 1nsk,L,2.800,3bbb,A,1.300
8 1t04,B,3.000,1t3f,B,2.000
9 1kiu,B,3.000,1ze3,H,1.840
10 1kv3,A,2.800,2q3z,A,2.000
11 1wqs,A,2.800,2iph,A,1.750
12 3gwp,A,2.900,3i16,A,2.000
13 1c0n,A,2.800,1kmj,A,2.000
14 2r8j,A,3.100,3oha,A,2.000
15 1z5s,C,3.010,2grn,B,1.800
16 3icq,B,3.200,3m1i,A,2.000
17 1nb3,I,2.800,3kfq,C,1.990
18 1otc,A,2.800,2i0q,A,1.910
19 1ue8,A,3.000,3b4x,A,1.940
20 1a4o,A,2.800,2o02,A,1.500
21 1t6i,A,2.810,3g4z,A,1.870
22 1de9,A,3.000,2o3h,A,1.900
23 2j6e,L,3.000,3n9g,L,1.434
24 3hrk,A,3.050,3lc0,A,1.800
25 2ecp,A,2.950,2asv,A,1.950
26 1z8l,A,3.500,2xeg,A,1.590
27 1ere,A,3.100,2r6w,A,2.000
28 1xni,A,2.800,3lh0,A,1.900
29 2pl0,A,2.800,3mpm,A,1.950
30 1pxt,A,2.800,1afw,A,1.800
31 1bmf,A,2.850,2jdi,A,1.900
32 2g3k,A,3.050,2j9u,A,2.000
33 3m9b,A,3.940,3m9h,A,2.000
34 2i0v,A,2.800,3dpk,A,1.950
35 1kiu,A,3.000,3bwu,C,1.760
36 1nys,A,3.050,2h62,D,1.850
37 1rgq,C,2.900,2o8m,C,2.000
38 1fg5,N,2.800,2vs4,A,1.770
39 1sj5,A,2.800,1vjl,A,1.900
40 2f7k,A,2.800,2yxt,A,2.000
41 1yrp,A,3.100,3bqr,A,1.750
... 82 3bqr
/net/chevy/raid1/nigel/amber/hi_lo
/net/chevy/raid1/nigel/amber/hi_lo/bq/3bqr
3bqr.pdb 4phenix_3bqr.pdb
local_archive None
archive PDB None
Reading from local mirror:
  /net/cci/pdb_mirror/pdb/bq/pdb3bqr.ent.gz
Model saved to /net/lewis/raid1/scratch1/nigel/amber/hi_lo/bq/3bqr/3bqr.pdb
Traceback (most recent call last):
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/build/../modules/cctbx_project/mmtbx/command_line/fetch_pdb.py", line 199, in <module>
    run2(sys.argv[1:])
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/build/../modules/cctbx_project/mmtbx/command_line/fetch_pdb.py", line 155, in run2
    files.append(get_pdb(id, data_type_, mirror, log, format=data_format))
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/modules/cctbx_project/iotbx/pdb/fetch.py", line 213, in get_pdb
    data = fetch(id, data_type, mirror=mirror, format=format, log=log)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/modules/cctbx_project/iotbx/pdb/fetch.py", line 150, in fetch
    data = libtbx.utils.urlopen(url)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/modules/cctbx_project/libtbx/utils.py", line 2069, in urlopen
    return urllib2.urlopen(*args, **kwds)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 127, in urlopen
    return _opener.open(url, data, timeout)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 404, in open
    response = self._open(req, data)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 422, in _open
    '_open', req)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 382, in _call_chain
    result = func(*args)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 1214, in http_open
    return self.do_open(httplib.HTTPConnection, req)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 1184, in do_open
    raise URLError(err)
urllib2.URLError: <urlopen error [Errno 110] Connection timed out>
local_archive None
archive PDB None
Reading from local mirror:
  /net/cci/pdb_mirror/pdb/bq/pdb3bqr.ent.gz
Model saved to /net/lewis/raid1/scratch1/nigel/amber/hi_lo/bq/3bqr/3bqr.pdb
Traceback (most recent call last):
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/build/../modules/cctbx_project/mmtbx/command_line/fetch_pdb.py", line 199, in <module>
    run2(sys.argv[1:])
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/build/../modules/cctbx_project/mmtbx/command_line/fetch_pdb.py", line 155, in run2
    files.append(get_pdb(id, data_type_, mirror, log, format=data_format))
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/modules/cctbx_project/iotbx/pdb/fetch.py", line 213, in get_pdb
    data = fetch(id, data_type, mirror=mirror, format=format, log=log)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/modules/cctbx_project/iotbx/pdb/fetch.py", line 150, in fetch
    data = libtbx.utils.urlopen(url)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/modules/cctbx_project/libtbx/utils.py", line 2069, in urlopen
    return urllib2.urlopen(*args, **kwds)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 127, in urlopen
    return _opener.open(url, data, timeout)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 404, in open
    response = self._open(req, data)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 422, in _open
    '_open', req)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 382, in _call_chain
    result = func(*args)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 1214, in http_open
    return self.do_open(httplib.HTTPConnection, req)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 1184, in do_open
    raise URLError(err)
urllib2.URLError: <urlopen error [Errno 110] Connection timed out>

==================================================
Summary of pdb4amber for: 3bqr.pdb
===================================================
REDUCE returned non-zero exit status: See reduce_info.log for more details

----------Chains
The following (original) chains have been found:
A

---------- Alternate Locations (Original Residues!))

The following residues had alternate locations:
ILE_20
ARG_22
SER_44
ARG_57
ARG_66
LYS_115
ASP_119
GLU_137
MET_140
LYS_213
SER_221
SER_260
HOH_373
HOH_429

---------- Gaps (Renumbered Residues!)
gap of 5.845013 A between ASN 166 and ASN 167

---------- Mising heavy atom(s)

ARG_42 misses 4 heavy atom(s)
ARG_47 misses 3 heavy atom(s)
LYS_80 misses 3 heavy atom(s)
GLN_112 misses 3 heavy atom(s)
GLU_163 misses 4 heavy atom(s)
ASN_166 misses 3 heavy atom(s)
ASN_167 misses 3 heavy atom(s)
ILE_168 misses 3 heavy atom(s)
LYS_213 misses 3 heavy atom(s)
LYS_252 misses 2 heavy atom(s)
ARG_271 misses 6 heavy atom(s)
ARG_272 misses 6 heavy atom(s)
ARG_275 misses 6 heavy atom(s)
The alternate coordinates have been discarded.
Only the first occurrence for each atom was kept.
not prmtop_filename /net/lewis/scratch1/nigel/amber/hi_lo/bq/3bqr/4amber_3bqr.prmtop
not prmtop_filename /net/chevy/raid1/nigel/amber/hi_lo/bq/3bqr/4amber_3bqr.prmtop

	Running AmberPrep : 3bqr.pdb


==================================================
Running pdb4amber on 3bqr.pdb
==================================================

==================================================
Setting up library files for non-standard residues
==================================================

  Residue "PGO" already in amber monomer library. Skipping elbow/antechamber
    run for this residue.
        

  Residue "GOL" already in amber monomer library. Skipping elbow/antechamber
    run for this residue.
        

==================================================
Preparing asu files and 4phenix_3bqr.pdb
==================================================

~> tleap -f 3bqr_asu_tleap_input_run

Checking output filenames
  file : 3bqr_asu.prmtop
Sorry:   Output file is empty : 3bqr_asu.prmtop
  Check contents of "leap.log"
